libraries/ESP8266HTTPClient/src/ESP8266HTTPClient.cpp

```diff
@@ -27,8 +27,9 @@
 
 #ifdef HTTPCLIENT_1_1_COMPATIBLE
 #include <ESP8266WiFi.h>
-#include <WiFiClientSecure.h>
+#include <WiFiClientSecureAxTLS.h>
```

Comment:
Something doesn't look right in this file with the HTTPCLIENT_1_1_COMPATIBLE define. Did you test both defined and undefined cases?

Reply:
This patch is now reverted, axTLS is however explicitely mentioned in this file.
I just fixed the deprecation, nothing more.